pages/common/luac.md

```diff
@@ -0,0 +1,11 @@
+# luac
+
+> Lua byte code compiler.
+
+- Compile a lua source code to a lua byte code:
+
+`luac -o {{byte_code.luac}} {{source.lua}}`
+
+- Don't include a debug information in the byte code:
```

Comment:
Perhaps `Don't include debug information in the output` would read better?

Reply:
Ok, but the article should probably stay there. So that it's grammatically correct. What do you think?

Reply:
'information' is not a countable noun, so "a debug information" is incorrect.